### ISSUE TITLE
Details about the unsup-NMT pretraining?

### ISSUE THREAD

**Issue by LibertFan** [microsoft/MASS#126]
Could you provide some details, learning_rate, batch_size, update_freq, GPU_NUM, etc., about the unsup-NMT for en-fr, en-de and en-ro? 

Is warmup needed here?
